taipy/gui/_renderers/factory.py

```diff
@@ -153,6 +154,25 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "time": lambda gui, control_type, attrs: _Builder(
```

Comment:
foe an easier life, we tend to sort components by name

Reply:
Ok, re-arranged